src/app/components/search/Search.jsx

```diff
@@ -4,20 +4,19 @@ import { useInput } from "../../hooks/useInput";
 import Magnifier from "../../icons/Magnifier";
 
 const Search = ({ saveSearch }) => {
-    const [search, setSearch] = useInput("");
-
-    const handleSubmit = e => {
-        e.preventDefault();
-        saveSearch(search.value);
+    const handleOnChange = value => {
+        saveSearch(value);
     };
 
-    const handleRest = () => {
+    const [search, setSearch] = useInput("", handleOnChange);
+
+    const handleReset = () => {
```

Comment:
Bit of an odd one this - VScode doesn't flag it, code compiles and works fine - but since it works either way i've removed the empty string arg for now